.storybook/stories/checkbox-toggle/checkbox-toggle-container.stories.ts

```diff
@@ -32,155 +32,75 @@ export default {
     type: { control: 'inline-radio', options: CheckboxType },
     createArray: { control: { disable: true }, table: { disable: true } },
     optionCount: { control: { type: 'number', min: 1, max: 100 } },
-    values: { control: { disable: true }, table: { disable: true } },
   },
   args: {
     // inputs
     clrInline: false,
     // story helpers
     type: CheckboxType.Checkbox,
     label: 'Options',
```

Comment:
Either set to `Option` or remove.

```suggestion
    label: 'Option',
```